src/main/java/com/rocketden/main/service/GameService.java

```diff
@@ -9,7 +8,6 @@
 import com.rocketden.main.exception.api.ApiException;
 import com.rocketden.main.model.Room;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.messaging.simp.SimpMessagingTemplate;
 import org.springframework.stereotype.Service;
 
 @Service
```

Comment:
Note: I think now that we have GameManagementService, this can probably be deleted (and the logic moved to GameManagementService)

Reply:
That's fair -- since there's likely a good number of changes I'll wait on more before I migrate that.

Reply:
Let me know if you want me to change that in this PR though, no problem if so.